core/src/cluster_info.rs

```diff
@@ -1656,6 +1717,7 @@ impl ClusterInfo {
                     if start - last_contact_info_trace > 10000 {
                         // Log contact info every 10 seconds
                         info!("\n{}", self.contact_info_trace());
+                        info!("\n{}", self.rpc_info_trace());
```

Comment:
mega-info:
```suggestion
                        info!("\n{}\n{}", self.contact_info_trace(), self.rpc_info_trace());
```
to avoid that `[2020-10-21T06:20:33.000864000Z INFO  solana_core::cluster_info]`?